src/common/queries/tax-rates.ts

```diff
@@ -51,3 +53,14 @@ export function bulk(
     ids: id,
   });
 }
+
+export function useBlankTaxRateQuery() {
+  return useQuery<TaxRate>(
+    '/api/v1/tax_rates/create',
+    () =>
+      request('GET', endpoint('/api/v1/tax_rates/create')).then(
+        (response: GenericSingleResourceResponse<TaxRate>) => response.data.data
+      ),
+    { staleTime: Infinity }
```

Comment:
Same for this. Is this query admin only? @turbo124

Reply:
@beganovich yes, admin only

Reply:
Condition made